java/maven.embedder/external/binariesembedded-list

```diff
@@ -17,44 +17,44 @@
 
 0235BA8B489512805AC13A8F9EA77A1CA5EBE3E8;org.aopalliance:aopalliance:1.0
 1194890E6F56EC29177673F2F12D0B8E627DEC98;org.apache.httpcomponents:httpclient:4.5.14
-7F2E0C573EAA7A74BAC2E89B359E1F73D92A0A1D;org.apache.httpcomponents:httpcore:4.5.15
+51CF043C87253C9F58B539C9F7E44C8894223850;org.apache.httpcomponents:httpcore:4.4.16
```

Comment:
@mbien @neilcsmith-net sorry this is an issue, it will generate a pom with 4.5.15 as hash is correct it will map to the maven artefacts no brain. Only way to figure out is on vote :/ this is late. 

maybe the binariesembedded-list should be used to download the way binaris-list too to check that in the future

Reply:
We really need to generate this file from the zip in the build! :smile:

I'm not sure I understand @ebarboni what the implications are here?  Will this only show as a problem on release or will it show up in snapshots?  Do we need to fix in delivery?

Reply:
opened a PR for this #5939 in case we have to fix it. I didn't check the rest of the versions - this was just a coincidence that I saw it.

Reply:
It's release critical (for platform maven users), and good findings to get it before release. 
pom from org-netbeans-modules-maven-embedder-RELEASE180 contains that (from zip in ci build)
<dependency>
      <groupId>org.apache.httpcomponents</groupId>
      <artifactId>httpcore</artifactId>
      <version>4.5.15</version>
      <classifier></classifier>
    </dependency>

and thats not good will lead to issue for user later after release. Because not everyone is using that dependency.
I fully aggree for additional tooling. 
Also may want to auto stage on release build if possible as it's apache / apache so we may have early test. (Snapshot are not exacly the same tree as the populating of snapshot is not limited to release cluster)

So additionnal build could have a integration test application to check all clusters pom.

Reply:
@ebarboni if you think this is release critical, please review #5939 then add the `priority:critical` label and we can look at building an 18-rc4 early next week.

Reply:
maybe restricting netbeans-tlp master build (release and not everything) so snapshot will be less and we could do IT on it.
I forget that staging require signing that implies human  and 30mins to sign every bits